src/modules/launcher/Plugins/Common/BrowserInfo.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO.Abstractions;
+using System.Text;
+using Wox.Plugin.Logger;
+
+/// <summary>
+/// Contains information (e.g. path to executable, name...) about the default browser.
+/// </summary>
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1060:Move pinvokes to native methods class", Justification = "We will refactor NativeMethods files later")]
+public class BrowserInfo
```

Comment:
Did you missed the namespace? 🤔

Reply:
is a namespace necessary?

Reply:
If you don't have one it uses the global one which is imo not a good idea. But I not so good in this.

Reply:
i think not using a namespace is simpler since we have to reference the file in the .csproj for each plugin anyway

Reply:
Namespace is preferable indeed. e.g. `PowerLauncher.Plugins.Common`